notebooks/pandas/raw/indexing-selecting-assigning.ipynb

```diff
@@ -118,6 +118,34 @@
     "desc"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%%RM_IF(PROD)%%\n",
+    "# Correct\n",
+    "desc = reviews[\"description\"]\n",
+    "\n",
+    "q1.check()\n",
+    "desc"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%%RM_IF(PROD)%%\n",
+    "# Correct\n",
```

Comment:
Rather than typing `# correct` and then using q1.check(), I would leave out the note `# correct` in these testing cells and use q1.assert_check_passed()

This will throw an exception if check() returns that the answer was incorrect.  The main advantages of assert_check_passed() is that if this ever starts failing (e.g. due to a change in pandas, a change in the underlying dataset, a change in the underlying checking code, whatever...) our CI tests will flag that this isn't returning Correct. Then we can fix it.

If you show `Correct` and visually verify that's the answer, it can only be tested manually. Which we don't do that frequently.

There are lots of places in the codebase where I've done like you did here, and those are just places I haven't updated since we added `assert_check_passed` and `assert_check_failed`.